### PR TITLE
Fix gamename module having newlines being converted into spaces

### LINKED ISSUE #1027: Extra space in Game Resources console pages
Minor text bug in the blue box at the top of each Game Resources [console page](https://tasvideos.org/GameResources/GBA). There is an extra space between the console name and the period; for example:
> This page documents information about Game Boy Advance .

This issue does not affect game pages like [GBA/AlienHominid](https://tasvideos.org/GameResources/GBA/AlienHominid).

### PR DESCRIPTION
**Pull request by Masterjun3**
Resolves #1027.

### CODE REVIEW

File: TASVideos/Pages/Shared/Components/GameName/Default.cshtml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ﻿@model IList<GameNameModel>
 @if (Model.Any(g => g.IsSystem))
-{
```
Comment (adelikat) [inline TASVideos/Pages/Shared/Components/GameName/Default.cshtml:3]: The "proper" way to fix issues like this in razor is to use the <text> tag:
@if (foo)
{
  <text>@SomeString</text>
}

**Review by adelikat — CHANGES_REQUESTED** [review]

File: TASVideos/Pages/Shared/Components/GameName/Default.cshtml (hunk 1 above)
Comment (Masterjun3) [reply TASVideos/Pages/Shared/Components/GameName/Default.cshtml:3]: What?

Comment (adelikat) [reply TASVideos/Pages/Shared/Components/GameName/Default.cshtml:3]: https://stackoverflow.com/questions/4969065/how-to-use-text-tag-in-mvc-3-razor

**Comment by Masterjun3** [comment]
Changes made.

**Review by adelikat — APPROVED** [review]

**Pull request merged**